### ISSUE TITLE
Distribution

### ISSUE THREAD

**Issue by Haltroy** [Haltroy/Mindbank#4]
We need to be able to distribute Mindbank easily for these platforms:
 - [x] Windows (x64 and ARM64) as Installer + uninstaller + update checks
 - [x] Android (already done since actions release it as APK, we only need an update check)
 - [x] Linux
   - [x] PKGBUILD for Arch-based systems (for both normal and git version)
   - [x] DEB packages (Debian-based)
   - [x] RPM packages (RedHat-based)
   - [x] Tutorial for how to build it for other distributions/from source

**Comment by Haltroy** [comment]
Working on this on [FluxionViewer](https://github.com/Haltroy/FluxionViewer/pull/4) right now. Once that is finished I can copy stuff from there to here and edit it for Mindbank.
